models/questions.js

```diff
@@ -0,0 +1,112 @@
+const { db, DataTypes } = require('../database/index.js');
+const models = require('./index.js');
+
+const Questions = db.define('questions', {
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    allowNull: false,
+    autoIncrement: true
+  },
+  product_id: {
+    type: DataTypes.INTEGER,
+    allowNull: false
+  },
+  body: {
+    type: DataTypes.TEXT,
+    allowNull: false
+  },
+  date_written: {
+    type: DataTypes.STRING(20),
+    allowNull: false
+  },
+  asker_name: {
+    type: DataTypes.STRING(50),
+    allowNull: false
+  },
+  asker_email: {
+    type: DataTypes.STRING(50)
+  },
+  reported: {
+    type: DataTypes.INTEGER
+  },
+  helpful: {
+    type: DataTypes.INTEGER
+  }
+}, {underscored: true, timestamps: false});
+
+Questions.associate = (models) => {
+  Questions.hasMany(models.answers)
+}
+
+module.exports = {
+  Questions: Questions,
```

Comment:
I think these imports I had you add are unnecessary at this point, but you'd know better than me.